.github/workflows/dafny_interop.yml

```diff
@@ -4,6 +4,21 @@
 name: Dafny Interoperability Test
 
 on:
+  workflow_call:
```

Comment:
Would it make more sense to add this target to the Manual CI workflow instead? Or perhaps at least as well?